modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -380,12 +386,529 @@ export default class Ecdsa {
       proof: proofShare,
     };
 
-    return signers;
+    return {
+      wShare,
+      kShare,
+    };
+  }
+
+  /**
+   * Perform multiplicitive-to-additive (MtA) share conversion with another signer.
+   * Connection 1.2 in https://lucid.app/lucidchart/7061785b-bc5c-4002-b546-3f4a3612fc62/edit?page=IAVmvYO4FvKc#
+   * @param {SignConvert} shares
+   * @returns {SignConvertRT}
```

Comment:
could benefit from a comment that signConvertStep1 and signConvertStep2 are not completed by the same party. maybe an overall comment on if we have SignerA starting first and SignerB going second, which functions would each of them call.